flink-java/src/test/java/org/apache/flink/api/java/io/TextInputFormatTest.java

```diff
@@ -90,7 +91,9 @@ public void testSimpleRead() {
 
 	@Test
 	public void testNestedFileRead() {
-		String[] dirs = new String[] {"tmp/first/", "tmp/second/"};
+		String tmpDirName = "tmp-" + UUID.randomUUID().toString().substring(0, 8);
```

Comment:
We could reuse `CommonTestUtils#createTempDirectory()'  in flink-runtime by copying the method into the CommonTestUtils class in `flink-test-utils-junit`.